packages/block-editor/src/components/url-input/style.scss

```diff
@@ -45,9 +45,9 @@ $input-size: 300px;
 
 	.components-spinner {
 		position: absolute;
-		right: $input-padding;
-		bottom: $input-padding + $grid-unit-10 + 1;
 		margin: 0;
+		top: calc(50% - #{$spinner-size} / 2);
```

Comment:
I would love for us to be able to remove as much spinner-positioning code across the codebase as we can. That's not a specific comment on this piece, which looks good. Just a general thought: it'd be nice if the component itself had generic good positioning in 90% of the cases without any additions.

Reply:
That would be great. I copied this style from the Snipper component.